smile_id_core/Utilities.py

```diff
@@ -31,14 +31,12 @@ def get_job_status(
         self,
         partner_params: Dict,
         option_params: Dict,
-        sec_params: Optional[Dict] = None,
+        signature: Optional[Dict] = None,
```

Comment:
This is a breaking API change. I think we should make it, we just need to talk about it in the changelog.